### ISSUE TITLE
API: comment on geokret require reference number. Maybe it shouldn't

### ISSUE THREAD

**Issue by kumy** [geokrety/geokrety-website#6]
[Imported from bitbuket#92]

For logging the geokret with API it is necesary to provide the secret reference number. Maybe we can skip asking for the reference number?

Issue based on a requests from opencaching.pl team.

**Comment by boly38** [comment]
@kumy could you complte with the following:
- confirm related endpoint url / controller
- confirm that without "secret reference", the only log type allowed is "comment" ?
 I mean "secret reference"stay mandatory for following actions: "dropped" or "grabed" or "met" or  "dipped"

**Comment by kumy** [comment]
Yep, only "comment" should be allowed without `tracking-code`, I would add "and if user is authenticated" (using session on website, or via api-token) - this is something that is taken into account in the `GKv2 API`. BTW, on GKv2 API, anonymous users will only be able read, no write anymore…

> confirm related endpoint url / controller

Hum, the only way to add comments that I know is the `ruchy.php` page. Is this what you want to know?

**Comment by kumy** [comment]
Shouldn't this type of log require captcha to prevent spam?
